static/js/util/util.js

```diff
@@ -273,15 +273,21 @@ export function getPreferredName(profile: Profile, last: boolean = true): string
 export function getLocation(profile: Profile, showState: boolean = true): string {
   let { country, state_or_territory, city } = profile;
   let subCountryLocation, countryLocation;
+  city = city ? `${city}, ` : "";
+
   if ( country === 'US' ) {
     let state = state_or_territory.replace(/^\D{2}-/, '');
-    subCountryLocation = showState ? `${city}, ${state}` : city;
+    subCountryLocation = showState ? `${city}${state}, ` : city;
     countryLocation = 'US';
+  } else if (country) {
+    subCountryLocation = city;
+    let countryFromCode = iso3166.country(country);
```

Comment:
In fact, given that `codeToCountryName` is now null-safe you should just be able to do collapse the `if else` and `else` blocks like this:

```js
else {
  subCountryLocation = city;
  countryLocation = codeToCountryName(country);
}
```